terraform/examples/io500/run_io500-sc21.sh

```diff
@@ -123,7 +123,7 @@ daos cont get-prop ${POOL_LABEL} ${CONT_LABEL}
 log "Use dfuse to mount ${CONT_LABEL} on ${IO500_RESULTS_DFUSE_DIR}"
 pdsh -w ^hosts sudo rm -rf "${IO500_RESULTS_DFUSE_DIR}"
 pdsh -w ^hosts mkdir -p "${IO500_RESULTS_DFUSE_DIR}"
-pdsh -w ^hosts dfuse --pool="${POOL_LABEL}" --container="${CONT_LABEL}" --mountpoint="${IO500_RESULTS_DFUSE_DIR}"
+pdsh -w ^hosts dfuse -S --pool="${POOL_LABEL}" --container="${CONT_LABEL}" --mountpoint="${IO500_RESULTS_DFUSE_DIR}"
```

Comment:
@johannlombardi  Does this change the perf per client in GCP?

Reply:
Not for io500 since it does not use dfuse. For app using dfuse directly, -S should not be used.